harmony-api/src/services/userService.ts

```diff
@@ -2,6 +2,18 @@ import * as userRepo from '../repos/userRepo'
 import { User } from '../models/user'
 import { HttpError } from '../models/error/httpError'
 
+export const get = async (
```

Comment:
Also in the future could we use a more descriptive names once we get past the controller/router? `get` makes sense there since its the name of the httpMethod, but something like `getUserByUsername` might make more sense in the service